pkg/session_recording/recordings.jsx

```diff
@@ -331,6 +418,10 @@
                          j--);
                     recordingList.splice(j + 1, 0, r);
                 } else {
+                    /* Remove existing recording for list if it's not in correct timeframe */
+                    if(ts < r.start) {
+                        delete this.recordingMap[id];
+                    }
```

Comment:
No, it's not needed, I think it was just a mistake in commit. And in addition to this such usage of delete for array element should be avoided, because it will leave like an empty slot, so it was a bad idea to use delete here.